azure_jumpstart_ag/bicep/data/dataExplorer.bicep

```diff
@@ -16,35 +16,68 @@ param skuName string = 'Dev(No SLA)_Standard_E2a_v4'
 param skuTier string = 'Basic'
 
 @description('The name of the Azure Data Explorer POS database')
-param posOrdersDBName string = 'posOrders'
-
-@description('The GUID used for naming')
-param namingGuid string
+param posOrdersDBName string
 
 @description('The ID of the IoT Hub')
 param iotHubId string
 
 @description('The name of the IoT Hub consumer group')
 param iotHubConsumerGroup string
 
-resource adx 'Microsoft.Kusto/clusters@2022-12-29' = {
-  name: ClusterName
+@description('The Naem of the Cosmos DB account')
+param cosmosDBAccountName string
+
+@description('# of nodes')
+@minValue(1)
+@maxValue(2)
+param skuCapacity int = 1
+
+//  Id of the Cosmos DB data reader role
+var cosmosDataReader = '00000000-0000-0000-0000-000000000001'
+
+resource cosmosDBAccount 'Microsoft.DocumentDB/databaseAccounts@2023-03-01-preview' existing = {
+  name: cosmosDBAccountName
+}
+
+resource adxCluster 'Microsoft.Kusto/clusters@2022-12-29' = {
+  name: adxClusterName
   location: location
   tags: resourceTags
   sku: {
     name: skuName
     tier: skuTier
+    capacity: skuCapacity
+  }
+  
+  // Assign system assigned identity
+  identity: {
+    type: 'SystemAssigned'
+  }
+
+  // Wait until Cosmos DB account is created to assign role permissions and create connection
+  dependsOn: [
+    cosmosDBAccount
+  ]
+}
+
+resource ordersScript 'Microsoft.Kusto/clusters/databases/scripts@2022-12-29' = {
+  name: 'ordersScrit'
+  parent: posOrdersDB
+  properties: {
+    continueOnErrors: false
+    forceUpdateTag: 'string'
+    scriptContent: loadTextContent('script.kql')
   }
 }
 
 resource posOrdersDB 'Microsoft.Kusto/clusters/databases@2022-12-29' = {
-  parent: adx
+  parent: adxCluster
   name: posOrdersDBName
   location: location
   kind: 'ReadWrite'
 }
 
-resource dxdatabaseConnection 'Microsoft.Kusto/clusters/databases/dataConnections@2022-12-29' = {
+resource dxdatabaseIotConnection 'Microsoft.Kusto/clusters/databases/dataConnections@2022-12-29' = {
```

Comment:
is dx here data explorer? If so, can we be consistent with using adx instead of dx for variable names?

Reply:
Fixed.